wp-dependency-installer.php

```diff
@@ -585,39 +614,42 @@ public function admin_notices() {
 				return false;
 			}
 			foreach ( $this->notices as $notice ) {
-				$status  = empty( $notice['status'] ) ? 'updated' : $notice['status'];
-				$message = empty( $notice['message'] ) ? '' : esc_html( $notice['message'] );
-
-				if ( ! empty( $notice['action'] ) ) {
-					$action   = esc_attr( $notice['action'] );
-					$message .= ' <a href="javascript:;" class="wpdi-button" data-action="' . $action . '" data-slug="' . $notice['slug'] . '">' . ucfirst( $action ) . ' Now &raquo;</a>';
+				$status      = isset( $notice['status'] ) ? $notice['status'] : 'updated';
```

Comment:
Prefer `isset()` instead of `empty()`, so third-party devs can also use the following parameter:

```php
$notice['status'] = '';
```

**Before:**
```html
<div class="updated notice is-dismissible">blah blah blah</div>
```

**After:**

```html
<div class="notice is-dismissible">blah blah blah</div>
```